pom.xml

```diff
@@ -176,6 +176,12 @@
         <jodah-failsafe.version>0.9.5</jodah-failsafe.version>
         <apache.shiro.version>1.3.2</apache.shiro.version>
         <spark.version>2.5.5</spark.version>
+
+        <!-- gitflow-incremental-builder -->
+        <!-- See https://github.com/vackosar/gitflow-incremental-builder for a list of all the options -->
+        <gib.referenceBranch>refs/remotes/origin/master</gib.referenceBranch>
```

Comment:
I almost never use origin :-) usually i have my remotes named for the org or user that the fork belongs to (i like to be able to see where i'm pushing immediately). It looks like this would be fine though since the reference branch property has the remote name in it